TuneUp/TuneUpViewExtension.cs

```diff
@@ -18,7 +18,6 @@ public class TuneUpViewExtension : ViewExtensionBase, IViewExtension
 
         public override void Dispose()
         {
-            TuneUpView.Dispose();
```

Comment:
Any reason to removing the dispose on the view when the extension is closed?

Reply:
AFAICT it was only unsubscribing from DynamoWindow_SizeChanged, which is not needed anymore.